janitor/__init__.py

```diff
@@ -20,4 +29,4 @@ def get_features_targets(*args, **kwargs):
     return _get_features_targets(*args, **kwargs)
 
 
-__version__ = "0.23.1"
+__version__ = "0.23.1-lazy"
```

Comment:
I'm unsure whether adding a suffix will have an impact on `bumpversion`.
Once the CI of `auto-release.yml` is trigged, the version will be like `0.24.0` or `0.24.0-lazy`?

Reply:
suffix -lazy to __version__ value  was introduced by me since I did't know what to do about version in the case 
of PR branch.
so feel free to drop it and set __version__ appropriately to current situation

Reply:
I'd like to reverse this change to version first.

Reply:
I agree with you, @Zeroto521, very astute catch that I didn't see before! @asmirnov69 could you remove the `-lazy` here? We'll take care of versioning in our build pipeline.

Reply:
will do:
- __verision__ will be 0.23.1
- lazy_loader will be added to .requirements/base.in

Reply:
done